src/lib/core/components/VariableList.tsx

```diff
@@ -159,8 +172,29 @@ export default function VariableList(props: VariableListProps) {
     [getFieldSearchString]
   );
 
+  const availableVariables = useMemo(() => {
+    const availableVariablesArray = getLeaves(
+      fieldTree,
+      (node) => node.children
+    ).map((node) => node.field.term.split('/')[1]);
+
+    return new Set(availableVariablesArray);
+  }, [fieldTree]);
```

Comment:
We shouldn't use `getLeaves` here. There are cases where variables can have children. The [`isFilterField`](https://github.com/VEuPathDB/WDKClient/blob/master/Client/src/Components/AttributeFilter/AttributeFilterUtils.ts#L30) function can be used to find all variables.

Reply:
Good catch! PR updated.